### ISSUE TITLE
question

### ISSUE THREAD

**Issue by Mesverrum** [kentik/ktranslate#545]
@314 Looking at #350 I see the example of an '=' but are there other operators I can leverage, for example if I wanted to do a like '*something*' or in(a,b,c) would it be possible to do?

**Comment by Mesverrum** [comment]
https://github.com/kentik/ktranslate/blob/9cf390bb2fe675d6d84ea7a209bb3db296451972/pkg/inputs/snmp/metrics/device_metrics.go#L429 that leads me to believe we only support exact matches, yes?

**Comment by i3149** [comment]
You are right in your hunch that right now only `=` will work. Wouldn't be that hard to extend though to other operators.
